examples/shader/post_processing.rs

```diff
@@ -0,0 +1,265 @@
+//! A custom post processing effect, using two render passes and a custom shader.
+//! Here a chromatic aberration is applied to a 3d scene containting a rotating cube.
+//! This example is useful to implement your own post-processing effect such as
+//! edge detection, blur, pixelization, vignette... and countless others.
+
+use bevy::{
+    core_pipeline::clear_color::ClearColorConfig,
+    ecs::system::{lifetimeless::SRes, SystemParamItem},
+    prelude::*,
+    reflect::TypeUuid,
+    render::{
+        camera::{Camera, RenderTarget},
+        render_asset::{PrepareAssetError, RenderAsset, RenderAssets},
+        render_graph::{Node, NodeRunError, RenderGraph, RenderGraphContext, SlotValue},
+        render_resource::{
+            BindGroup, BindGroupDescriptor, BindGroupEntry, BindGroupLayout,
+            BindGroupLayoutDescriptor, BindGroupLayoutEntry, BindingResource, BindingType,
+            Extent3d, SamplerBindingType, ShaderStages, TextureDescriptor, TextureDimension,
+            TextureFormat, TextureSampleType, TextureUsages, TextureViewDimension,
+        },
+        renderer::{RenderContext, RenderDevice},
+        view::RenderLayers,
+        RenderApp,
+    },
+    sprite::{Material2d, Material2dPipeline, Material2dPlugin, MaterialMesh2dBundle},
+};
+
+#[derive(Component, Default)]
+pub struct PostProcessingPassCamera;
+
+/// The name of the final node of the post process pass.
+pub const POST_PROCESS_PASS_DRIVER: &str = "post_process_pass_driver";
```

Comment:
not used ? also, "main pass" and "post processing" pass might be disorienting for readers, as due to https://github.com/bevyengine/bevy/pull/4745, this "post processing" implementation uses camera priority rather than excessive rendergraph node implementation.